codegen/src/generator/rest_json.rs

```diff
@@ -115,6 +115,78 @@ impl GenerateProtocol for RestJsonGenerator {
     }
 }
 
+fn http_code_to_status_code(code: Option<i32>) -> String {
+    match code {
+        Some(actual_code) => {
+            // From http://hyper.rs/hyper/v0.9.12/src/hyper/src/status.rs.html and modified to return strings:
```

Comment:
I tried using the `Display` trait [StatusCode implements](https://docs.rs/reqwest/0.2.0/reqwest/enum.StatusCode.html) but it prints `the status code, including the canonical reason.`  EG: `assert_eq!(format!("{}", ImATeapot), "418 I'm a teapot");`.  Perhaps I can try implementing `Display` on an enum that derives from Debug as suggested.